topydo/lib/Colors.py

```diff
@@ -0,0 +1,70 @@
+# Topydo - A todo.txt client written in Python.
+# Copyright (C) 2014 - 2015 Bram Schoenmakers <me@bramschoenmakers.nl>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" This module serves for managing output colors. """
+
+from topydo.lib.Config import config
+
+class Colors(object):
+    def __init__(self):
+        self.priority_colors = config().priority_colors()
+        self.project_color = config().project_color()
+        self.context_color = config().context_color()
+        self.metadata_color = config().metadata_color()
+        self.link_color = config().link_color()
+
+    def _int_to_ansi(self, p_int, decoration):
```

Comment:
decoration should have default value 0, also please rename to `p_decorator` to be consistent with the rest of the code base.